DocSum/docker/gaudi/README.md

```diff
@@ -44,12 +44,26 @@ cd GenAIExamples/DocSum/docker/ui/
 docker build -t opea/docsum-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f ./docker/Dockerfile .
 ```
 
+### 5. Build React UI Docker Image
+
+Build the frontend Docker image via below command:
+
+```bash
+cd GenAIExamples/DocSum/docker/ui/
+export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/docsum"
+docker build -t opea/docsum-react-ui:latest --build-arg BACKEND_SERVICE_ENDPOINT=$BACKEND_SERVICE_ENDPOINT -f ./docker/Dockerfile.react .
+```
+
+Note:
+If docker build fails, update the Base image in `Dockerfile.react` to `node:20.10.0-alpine`
```

Comment:
we observed the same tsc not found issue in gaudi build . so we added this note to update the base image . Please try it with this base image